efb_telegram_master/slave_message.py

```diff
@@ -993,7 +995,7 @@ def generate_message_template(self, msg: Message, singly_linked: bool) -> str:
                 name_prefix += f", {msg.author.long_name}"
             msg_template = f"{emoji_prefix} {name_prefix}:"
         else:
-            msg_template = f"{Emoji.UNKNOWN} {msg.author.long_name} ({msg.chat.display_name}):"
+            msg_template = f"{Emoji.UNKNOWN} {msg.author.long_name} {'(' + msg.chat.display_name + ')' if msg.chat.display_name else ''}:"
```

Comment:
`msg.author.long_name` already contains the display name. It should be safe to drop the second part.

```suggestion
            msg_template = f"{Emoji.UNKNOWN} {msg.author.long_name}:"
```

Reply:
You are right, the changes here are indeed wrong.
![image](https://user-images.githubusercontent.com/49866772/206397821-c141d480-913b-4bac-8b88-a683ebdbb246.png)
I think I should amend this here. Before the change, the empty brackets bug was also occasional and could be fixed with a restart.
![image](https://user-images.githubusercontent.com/49866772/206398869-e4553640-2ef4-465f-9673-38156c7a994d.png)
 I'm not sure exactly why. But after fixing it with the image above, the empty brackets have not occurred again.

Reply:
I have submitted pr to efb, you can see https://github.com/ehForwarderBot/ehForwarderBot/pull/312.

Reply:
Moving discussion over to https://github.com/ehForwarderBot/ehForwarderBot/pull/312.